Secrets with Cargo Workspaces (#260)
shuttle-hq/shuttle-docs: resources/shuttle-secrets.mdx

  with no way of providing them in code. Sometimes, this can be solved by
  manually setting the variable after loading the secret (and before loading the library):
  `std::env::set_var("SOME_ENV_VAR", my_secret);`

> - If you are using a [Cargo Workspace](configuration/project-structure#workspaces} you will need to ensure your `Secrets.dev.toml` file and `Secrets.toml`

Reviewer: ```suggestion
- If you are using a [Cargo Workspace](/configuration/project-structure#workspaces) you will need to ensure your `Secrets.dev.toml` or `Secrets.toml` file is located in the crate with the `shuttle_runtime::main` macro.
```

>     files are located in the crate with the `shuttle_runtime::main macro.`

Reviewer: ```suggestion

```
